website/content/docs/plugins/creation/index.mdx

```diff
@@ -247,21 +247,6 @@ To include plugin docs on Packer.io a global pre-hook has been added to the main
 
 The `docs.zip` file will contain all of the `.mdx` files under the plugins root `docs/` directory that can be consumed remotely by Packer.io.
 
-Once the first `docs.zip` file has been included into a release you will need to open a one time pull-request against [hashicorp/packer](https://github.com/hashicorp/packer) to register the plugin docs.
-
-This is done by adding the block below for the respective plugin to the file [website/data/docs-remote-navigation.js](https://github.com/hashicorp/packer/blob/master/website/data/docs-remote-plugins.json).
-
-```json
-{
-   "title": "Scaffolding",
-   "path": "scaffolding",
-   "repo": "hashicorp/packer-plugin-scaffolding",
-   "version": "latest"
- }
-```
-
-If a plugin maintainer wishes to only include a specific version of released docs then the `"version"` key in the above configuration should be set to a released version of the plugin. Otherwise it should be set to "latest".
-
```

Comment:
I noticed this content was duplicated between tabs, and in both cases, occurred at the end of the tabbed content. I've removed the duplication by adding this content once, after the closing `<Tabs />`.

The [appearance after this change](https://packer-git-fork-zchsh-zsfix-plugin-latest-edit-links-hashicorp.vercel.app/docs/plugins/creation#registering-plugin-documentation) is identical in content flow and styling compared to [before this change](https://www.packer.io/docs/plugins/creation#registering-plugin-documentation).

Hopefully this is a helpful change, if there was another reason this content was duplicated, please let me know 🙇